Dockerfile

```diff
@@ -32,86 +32,44 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM debian:stretch
-
+FROM buildpack-deps:stretch AS base
```

Comment:
Can we start with `golang` image (which is essentially stretch with golang installed)?

Reply:
I think we could, cc @cpuguy83 👼

Reply:
cc @tianon 

I know this was attempted in the past, I don't recall why we didn't do it.

Reply:
If I recall correctly, at the time we were still using utilities that Ubuntu had better packaging for than Debian, so given we're on Debian now, it's likely OK. :+1:

Reply:
yay :tada:

Reply:
See https://github.com/moby/moby/pull/36425 👼